test/vinyl-luatest/update_optimize_test.lua

```diff
@@ -0,0 +1,530 @@
+local t = require('luatest')
+
+local server = require('test.luatest_helpers.server')
+
+local g = t.group()
+
+local dump_stmt_count_str = [[function dump_stmt_count(indexes)
+    local dumped_count = 0
+    for _, i in ipairs(indexes) do
+        dumped_count = dumped_count +
+            box.space.test.index[i]:stat().disk.dump.output.rows
+    end
+    return dumped_count
+end]]
+
+g.before_all(function()
+    local box_cfg = require('test.vinyl-luatest.box_cfg')
```

Comment:
This looks weird Lua-wise IMO. Let's instead name this file `common.lua` and put the default config in `common.default_box_cfg`.

Also, I'd make `default_box_cfg` a function so that a test could update the config in place, for example:

```lua
local common = require('test.vinyl.common')
local box_cfg = common.default_box_cfg()
box_cfg.vinyl_cache = 1024 * 1024
```